src/Parser/DecimalMoneyParser.php

```diff
@@ -47,7 +56,7 @@ public function parse($money, $forceCurrency = null)
          */
         $currency = $forceCurrency;
         if (!$currency instanceof Currency) {
-            @trigger_error('Passing a currency as string is deprecated since 3.1 and will be removed in 4.0. Please pass a '.Currency::class.' instance instead.', E_USER_DEPRECATED);
```

Comment:
Begone, hack that makes everything so much messier @_@ 

Heck, we may honestly just get rid of it overall, if we decide to break BC